mlos_bench/mlos_bench/tests/config/services/remote/mock/mock_vm_service.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+    "class": "mlos_bench.tests.services.remote.mock.mock_vm_service.MockVMService",
+    "config": {
+        "vmName": "test-vm"
```

Comment:
Could consider to have this get specified in the global config instead to reduce repeats, but this makes it standalone usable for now.